packages/core/src/PlayerContextProvider.js

```diff
@@ -70,15 +70,21 @@ const defaultState = {
 };
 
 // assumes playlist is valid
-function getGoToTrackState(prevState, index, shouldPlay = true) {
+function getGoToTrackState(
+  prevState,
+  index,
+  shouldPlay = true,
+  shouldForceLoad = false
+) {
   const isNewTrack = prevState.activeTrackIndex !== index;
   return {
     activeTrackIndex: index,
     trackLoading: isNewTrack,
     currentTime: 0,
     loop: isNewTrack ? false : prevState.loop,
```

Comment:
We didn't talk about this, but do you think we can we make this `(isNewTrack || shouldForceLoad) ? false : prevState.loop`? Don't think it hurts anything.